src/main/java/com/hotels/beeju/ThriftHiveMetaStoreJUnitRule.java

```diff
@@ -76,65 +62,28 @@ public ThriftHiveMetaStoreJUnitRule(String databaseName, Map<String, String> con
 
   @Override
   protected void beforeTest() throws Throwable {
-    thriftPort = -1;
-    startThrift();
+    thriftHiveMetaStoreCore.startThrift(thriftServer);
```

Comment:
can you rename the method on the core to `before`? to be more consistent with the delegation in the above rules?